torch/onnx/_internal/exporter/_onnx_program.py

```diff
@@ -117,31 +119,35 @@ def _create_value_mapping(graph: ir.Graph) -> dict[str, ir.Value]:
     return values
 
 
-def _to_ort_value(tensor: torch.Tensor) -> ort.OrtValue:
+def _to_ort_value(input: torch.Tensor | int | float | str | bool) -> ort.OrtValue:
     """Convert a PyTorch tensor to an ONNX Runtime OrtValue."""
     import onnxruntime as ort
 
     from torch.onnx._internal.exporter import _core
 
-    if tensor.dtype == torch.bfloat16 or tensor.dtype in _NP_UNSUPPORTED_DTYPES_8BIT:
+    if isinstance(input, (int, float, str, bool)):
+        # Convert scalar values to OrtValue
+        return ort.OrtValue.ortvalue_from_numpy(np.array(input))
```

Comment:
For float values should be make sure it is float32 (not float64)?

Reply:
Also int64 not int32

Reply:
You mean when I wrap it to numpy?